pkgs/tools/backup/gyb/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, fetchFromGitHub
+, python3
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "gyb";
+  version = "1.62";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "GAM-team";
+    repo = "got-your-back";
+    rev = "v${version}";
+    sha256 = "sha256-HaexQ0y5i9Q0xgjzAX6E2xeyeDvARo7/Gx3ytohRT7U=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    google-api-python-client
+    google-auth
+    google-auth-oauthlib
+    google-auth-httplib2
+    httplib2
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,${python3.sitePackages}}
+    mv gyb.py "$out/bin/gyb"
+    mv *.py "$out/${python3.sitePackages}/"
```

Comment:
This one is kinda wonky, but since its `buildPythonApplication` and not `buildPythonModule` it should be fine in my opinion. I would however like a second opinion though.

Reply:
@austinbutler You might could ask upstream if they could add a simple setup.py for installation